### DISCUSSION TITLE
Once Vim is opened, the following error message will appear

### DISCUSSION THREAD

**Discussion by wxzvim** [ayamir/nvimdots#859] [Q&A] [answered]
![image](https://github.com/ayamir/nvimdots/assets/110797126/a5317418-eb67-489f-8b84-f27c7a5ca9d0)

**Comment by charliie-dev** [comment]
Seems like your plugins are having problems during installation, remove the plugin folder and reopen nvim again.

**Reply by wxzvim** [reply]
![image](https://github.com/ayamir/nvimdots/assets/110797126/f41ff2b0-5b80-4272-882c-21be4afcc891)
update plugins, the above the error occurred.

**Reply by Jint-lzxy** [reply]
```sh
rm -rf ~/.local/share/nvim/site/lazy/
nvim
```

**Reply by wxzvim** [reply]
OK，but 
The following error has occurred again
![image](https://github.com/ayamir/nvimdots/assets/110797126/ed70381c-fe1f-457f-9242-34837d5f598b)

**Reply by Jint-lzxy** [reply]
For a quick fix use Neovim 0.9 or higher (and the `main` branch). Branch `0.8` hasn't been updated yet due to issues with Lspsaga v0.3. If you insist on using Neovim 0.8, please do `:Lazy restore`. _(\*Note: The `0.8` branch only supports plugins specified in `lazy-lock.json`)_

**Reply by Jint-lzxy** [reply, marked as answer]
Just updated the `0.8` branch. Pull the latest commit and u'll be good to go :)
